spectre-meltdown-checker.sh

```diff
@@ -695,6 +699,19 @@ check_variant2()
 				fi
 			done
 		fi
+		if [ "$ibrs_supported" != 1 ]; then
+			if grep -qi spec_ctrl /proc/cpuinfo; then
+				pstatus green YES
+				ibrs_supported=1
+			fi
+		fi
+		if [ "$ibrs_supported" != 1 -o "$ibrs_enabled" != 1 ]; then
+			if dmesg | grep -qi "Disabling indirect branch predictor support"; then
```

Comment:
It's not clear to me in which case exactly this message appears in the OpenSUSE backport, can you explain? (if you have a link to some source code, we can even document it)

Reply:
I looked over our kernel source code again, and I think i cannot use this message reliably.  :/

Reply:
I expect the microcode check part to work everywhere as is (with the `dd`), as the new MSR Intel introduced doesn't depend on the way kernels are backported at all.
However to check if IBRS is enabled, that's a different story... and I want to be extremely certain of the checks we add there :)